README.md

```diff
@@ -269,6 +269,8 @@ Where `wf_options.json` would contain:
 ```
 {
   "jes_gcs_root": "gs://my-bucket/workflows"
+  "account_name": "my.google.account@gmail.com"
+  "refresh_token": "1/Fjf8gfJr5fdfNf9dk26fdn23FDm4x"
```

Comment:
Yes and no. They are required only if in RefreshToken mode. Otherwise they're ignored.
There is a `JES Backend` section in the README which is currently empty and should definitely indicate that, among other stuff about JES that's not currently documented